dronekit/__init__.py

```diff
@@ -184,6 +184,13 @@ def __init__(self, north, east, down):
     def __str__(self):
         return "LocationLocal:north=%s,east=%s,down=%s" % (self.north, self.east, self.down)
 
+    def distance(self):
+        """
+        Distance away from home, in meters.
+        """
+        if self.north is not None and self.east is not None and self.down is not None:
```

Comment:
makes sense, but perhaps only break it up between horizontal and vertical? I don't think we want to report a distance if it's only North or only East. I'll push a new commit